api/base/settings/defaults.py

```diff
@@ -51,21 +51,24 @@
 }
 
 REST_FRAMEWORK = {
-    'PAGE_SIZE': 10,
-
-    # Order is important here because of a bug in rest_framework_swagger. For now,
```

Comment:
Even though the swagger docs aren't working, while we have django-rest-swagger in, we should keep the comment. If we remove django-rest-swagger, we can take this comment out.